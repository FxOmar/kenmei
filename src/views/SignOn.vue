<template lang="pug">
  .container.mx-auto.w-full.h-full
    .flex.flex-col.h-full.w-full.items-center.justify-center
      base-card.max-w-xs
        .px-6.py-4#sign-on-card
          component(
            :is='activeComponent'
            @componentChanged='activeComponent = $event'
          )
</template>

<script>
  import TheSignOnTabs from '@/components/TheSignOnTabs';
  import TheResetPassword from '@/components/TheResetPassword';

  export default {
    components: {
      TheSignOnTabs,
      TheResetPassword,
    },
    data() {
      return {
        activeComponent: 'TheSignOnTabs',
      };
    },
  };
</script>
