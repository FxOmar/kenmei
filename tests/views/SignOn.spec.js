import { shallowMount } from '@vue/test-utils';
import SignOn from '@/views/SignOn.vue';

describe('SignOn.vue', () => {
  it('renders the page', () => {
    expect(shallowMount(SignOn).element).toMatchSnapshot();
  });
});
