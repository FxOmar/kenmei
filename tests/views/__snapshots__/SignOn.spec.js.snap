// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignOn.vue renders the page 1`] = `
<div
  class="container mx-auto w-full h-full"
>
  <div
    class="flex flex-col h-full w-full items-center justify-center"
  >
    <base-card-stub
      class="max-w-xs"
    >
      <div
        class="px-6 py-4"
        id="sign-on-card"
      >
        <thesignontabs-stub />
      </div>
    </base-card-stub>
  </div>
</div>
`;
